src/core/viewBuilders/buildHelpMessage.ts

```diff
@@ -7,6 +7,7 @@ export function buildHelpMessage(channelId: string): ChatPostMessageArguments {
     text: `\
 Here are the available commands:
 
+- /homer changelog - Generate changelogs for any Gitlab project using release tags.
```

Comment:
What does that mean in practice? Is the changelog published somewhere?

Reply:
The changelog is displayed in a slack modal, nothing is published.

Do you want me to update the message to specify it ?